src/main/kotlin/com/blincke/commune_api/models/domain/friends/FriendshipState.kt

```diff
@@ -0,0 +1,10 @@
+package com.blincke.commune_api.models.domain.friends
+
+import com.blincke.commune_api.models.database.users.User
+
```

Comment:
Just meant to return the nature of a friendship, userXInitiated means userX is a requester and userY is a recipient